lib/config/fileconf_test.go

```diff
@@ -525,7 +516,10 @@ func TestX11Config(t *testing.T) {
 					"display_offset": "banana",
 				}
 			},
-			expectError: require.Error,
+			expectReadError: func(t require.TestingT, err error, i ...interface{}) {
```

Comment:
I wonder if this is worth testing, as it is not difficult to reason about the field type.

Same for similar tests.

Reply:
Agreed, removed these tests and the ones in `TestSSHSection`.